src/sshping.cxx

```diff
@@ -555,8 +654,7 @@ int run_upload_test(ssh_session ses) {
     ssh_scp_close(scp);
     ssh_scp_free(scp);
 
-    struct timespec t3;
-    clock_gettime(CLOCK_MONOTONIC, &t3);
+	uint64_t t3 = GetTime();
```

Comment:
nit indentation; yeah, you probably would have got this when final, but since I was here I'll call it out.
A couple other places too.  I trust you'll get 'em.